skt/executable.py

```diff
@@ -809,6 +850,13 @@ def check_args(parser, args):
        and not args.rh_configs_glob):
         parser.error("--cfgtype rh-configs requires --rh-configs-glob to set")
 
+    # Mail reporter required args: --mail-to and --mail-from
+    if (args._name == 'report' and args.type == 'mail'
+       and (not args.mail_to or not args.mail_from)):
+        parser.error(
+           "--reporter mail requires --mail-to and --mail-from to be set"
+        )
+
```

Comment:
It would be super useful to add a check for stdio reporter being called with mail reporter options -- the previous version threw an error about unexpected arguments, but doing `--reporter stdio --mail-to test` completes just fine now, which it IMHO shouldn't.